src/compose/ComposeBox.js

```diff
@@ -292,7 +292,9 @@ class ComposeBox extends PureComponent<Props, State> {
       alignItems: 'flex-end',
     },
     composeText: {
+      flex: 1,
       paddingVertical: 8,
+      justifyContent: 'flex-end',
```

Comment:
Have never imagined fix will be this short 🤦‍♂ 

Now I guess this solution is ideal as we wanted 😍